ray-operator/controllers/ray/common/pod.go

```diff
@@ -89,6 +89,7 @@ func DefaultHeadPodTemplate(instance rayiov1alpha1.RayCluster, headSpec rayiov1a
 	// TODO (Dmitri) The argument headPort is essentially unused;
 	// headPort is passed into setMissingRayStartParams but unused there for the head pod.
 	// To mitigate this awkwardness and reduce code redundancy, unify head and worker pod configuration logic.
+	log.Info("DefaultHeadPodTemplate called", "head envs ", headSpec.Envs)
```

Comment:
let's add some log level for this debug information?

Reply:
Will be removed later.